src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -245,13 +258,6 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
         bs.setString("event_manifest", m.eventManifest)
         bs.setBytes("event", m.serialized)
 
-        if (session.protocolVersion.compareTo(ProtocolVersion.V4) < 0) {
-          bs.setToNull("message")
-          (1 to maxTagsPerEvent).foreach(tagId => bs.setToNull("tag" + tagId))
-        } else {
-          bs.unset("message")
-        }
```

Comment:
The "message" is ancient and we still have it to be able to read old data, but we don't need to have it any of the write prepared statements, as far as I understand. Then we can have one stmt without tags and one with tags as it is now.